packages/front-end/components/Queries/ExpandableQuery.tsx

```diff
@@ -23,6 +28,7 @@ const ExpandableQuery: FC<{
     <div className="mb-4">
       <h4>
         {query.status === "running" && <FaCircle className="text-info mr-2" />}
+        {query.status === "queued" && <FaSquare className="text-info mr-2" />}
```

Comment:
I think keeping `FaCircle`, but changing from `text-info` to `text-secondary` would be clearer.  Might want to add a `title` prop here as well saying `Queued` so people can see what it means when they mouse over.